app/lib/i18n/fi.i18n.json

```diff
@@ -174,6 +174,8 @@
     "homes-header": "Yksikk\u00f6",
     "homes-header-addNewButton-group": "Lis\u00e4\u00e4 talo",
     "homes-homeGroup-addNewButton-home": "Lis\u00e4\u00e4 yksikk\u00f6",
+    "homes-homeGroup-assignManagerButton-home":"",
```

Comment:
We don't need to modify the fi.i18n.json directly, as that is done by our localization tool.